TestProjects/UniversalGraphicsTest/Assets/Scenes/057_SpeedTree_V7.unity

```diff
@@ -96,9 +96,11 @@ LightmapSettings:
     m_PVRFilteringAtrousPositionSigmaAO: 1
     m_ExportTrainingData: 0
     m_TrainingDataDestination: TrainingData
+    m_LightProbeSampleCountMultiplier: 4
   m_LightingDataAsset: {fileID: 112000002, guid: fd8f2c6386dc54f158818f3b4ce8a20d,
     type: 2}
-  m_UseShadowmask: 0
+  m_LightingSettings: {fileID: 4890085278179872738, guid: 42cfddced7968874e8459dfb9760f377,
```

Comment:
You added baked lighting to the 058_SpeedTree_V8 test scene. Do you have a scene without baked lighting to make sure that other code path keep working?

Reply:
> Why are there black artifacts on the leftmost tree in 057_SpeedTree_V7?

I think the scene is OK. The ref image looks darker around some leaves versus running the test scene in latest, looks like it just didn't trigger a new image output, presumably because it's within some error tolerance. We can force update it.

Reply:
> You added baked lighting to the 058_SpeedTree_V8 test scene. Do you have a scene without baked lighting to make sure that other code path keep working?

There are also non-baked lights in this scene on other trees.